site/content/en/docs/Getting Started/create-gameserver.md

```diff
@@ -31,7 +31,10 @@ For the purpose of this guide we're going to use the {{< ghlink href="examples/s
 Let's create a GameServer using the following command :
 
 ```
+# Linux Game Server
 kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/examples/simple-game-server/gameserver.yaml
+# Windows Game Server
```

Comment:
@markmandel - does that mean that we should assume that for getting started folks are primarily using linux VMs? If we go that route, we should add a note here with a link to the windows game server guide.

Reply:
Another thought is that folks may skip this getting started page completely and jump to https://agones.dev/site/docs/getting-started/create-fleet/ which says "While not required, you may wish to go through the Create a Game Server quickstart before this one." Rather than update both the game server and fleet getting started pages to have two paths, having them both link to a guide about windows game servers might be a better flow for users.